src/org/labkey/test/components/CustomizeView.java

```diff
@@ -443,6 +443,10 @@ public void addFilter(String[] fieldKeyParts, String column_name, String filter_
         if (!StringUtils.isBlank(filter))
         {
             WebElement valueInput = Locator.css(".item-value input").findElement(clauseRow);
+            if (filter_type.endsWith(" Of"))
```

Comment:
logic here based on wording of the filter seems fragile, somehow.
If we really just want to find a textArea if that's on offer, consider a locator that will find both?

`WebElement valueInput = Locator.XPathLocator.union(
                    Locator.tagWithClass("div","item-value").descendant("input"),
                    Locator.tagWithClass("div", "item-value").descendant("textArea")).findElement(clauseRow);`

(I haven't tested this but I expect it will work)

Reply:
I prefer finding by filterType because it also serves as additional test to check that textarea instead of text input is used for specific filter types.